tests/test_data_type.py

```diff
@@ -534,42 +534,42 @@ def test_path_api(self):
 
 class TestDatesetWrapper(TestBaseCase):
     def test_all(self):
-        data_set_warpper1 = DataSetWrapper(self.get_data_set())
-        data_set_warpper2 = DataSetWrapper(self.get_data_set())
+        data_set_wrapper1 = DataSetWrapper(self.get_data_set())
+        data_set_wrapper2 = DataSetWrapper(self.get_data_set())
```

Comment:
Sometimes it's hard to tell, especially in tests, but, probably?

Reply:
Nice catch.